src/Playwright.Tests/NetworkPostDataTests.cs

```diff
@@ -52,7 +52,7 @@ public async Task ShouldReturnCorrectPostdataBufferForUtf8Body()
 
             await Task.WhenAll(task, actualTask);
 
-            string expectedJsonValue = JsonSerializer.Serialize(value, new()
+            string expectedJsonValue = JsonSerializer.Serialize(value, new JsonSerializerOptions()
```

Comment:
Why are you bringing this back? Same here and in other places.

Reply:
the new() syntax fails because of a new overload to Serialize with 2 arguments.